Eatery/Controllers/Eateries/CampusMenuViewController.swift

```diff
@@ -156,6 +174,31 @@ class CampusMenuViewController: EateriesMenuViewController {
         }
     }
 
+    private func addExtendedMenuViewController() {
+        view.backgroundColor = .wash    // Changing background color here simplifies ScrollableViewController
+
+        if eatery.orderedExpandedCategories.count == 0 {
+            return
+        }
+
+        let expandedMenu = eatery.expandedMenu
+        var viewControllers: [CampusEateryExpandedMenuViewController] = []
+        var menuItems: [ExpandedMenu.Item] = []
+
+        for category in eatery.orderedExpandedCategories {
+            let menu = expandedMenu?.data[category] ?? []
+            let extendedMenuVC = CampusEateryExpandedMenuViewController(eatery: eatery, category: category, menu: menu)
+            viewControllers.append(extendedMenuVC)
+            menuItems.append(contentsOf: menu)
+        }
+
+        scrollableVC = ScrollableViewController(eatery: eatery, viewControllers: viewControllers, items: menuItems)
+        addChildViewController(scrollableVC!)
+        addToStackView(scrollableVC!.view)
+        scrollableVC!.didMove(toParentViewController: self)
+        scrollableVC!.view.hero.modifiers = createHeroModifiers(.fade, .translate)
```

Comment:
Can we avoid some of these force unwraps by using `let scrollableVC = ...` then assigning to self after?
```suggestion
        let scrollableVC = ScrollableViewController(eatery: eatery, viewControllers: viewControllers, items: menuItems)
        addChildViewController(scrollableVC)
        addToStackView(scrollableVC.view)
        scrollableVC.didMove(toParentViewController: self)
        scrollableVC.view.hero.modifiers = createHeroModifiers(.fade, .translate)
        self.scrollableVC = scrollableVC
```